hw/ip/otbn/dv/otbnsim/sim/constants.py

```diff
@@ -41,3 +41,20 @@ class ErrBits(IntEnum):
     LIFECYCLE_ESCALATION = 1 << 22
     FATAL_SOFTWARE = 1 << 23
     MASK = (1 << 24) - 1
+
+
+class LcTx(IntEnum):
+    r'''The same encoding as lc_tx_t in the RTL'''
```

Comment:
Why is the raw string literal required here?

Reply:
Just a documentation comment :-)